pkg/executor/core/executor.go

```diff
@@ -60,9 +61,17 @@ func (executor Executor) VerifySubject(ctx context.Context, verifyParameters e.V
 }
 
 func (executor Executor) GetVerifyRequestTimeout() time.Duration {
-	timeoutMilliSeconds := defaultRequestTimeoutMilliseconds
-	if executor.Config != nil && executor.Config.RequestTimeout != nil {
-		timeoutMilliSeconds = *executor.Config.RequestTimeout
+	timeoutMilliSeconds := defaultVerifyRequestTimeoutMilliseconds
+	if executor.Config != nil && executor.Config.VerificationRequestTimeout != nil {
+		timeoutMilliSeconds = *executor.Config.VerificationRequestTimeout
+	}
+	return time.Duration(timeoutMilliSeconds) * time.Millisecond
+}
+
+func (executor Executor) GetMutationRequestTimeout() time.Duration {
```

Comment:
Add tests for `GetVerifyRequestTimeout` and `GetMutationRequestTimeout`

Reply:
added unit tests for both